src/components/Table/index.tsx

```diff
@@ -1,2 +1,80 @@
-export * from './Table'
-export * from './interfaces'
+import React from 'react'
+
+type T = { [key: string]: any }
+
+interface Props {
+  tableClassName?: string
+  headerClassName?: string
+  columns: { key: string; label: string; accessor?: (row: T) => React.ReactNode }[]
+  data: T[]
+  actions?: { label: string; action: (row: T) => void }[]
+  getID: (row: T) => string
+  onRowClick?: (row: T) => void
+}
+
+// todo: actions header int'lize
+
+const Table = (props: Props) => {
+  const { tableClassName, headerClassName, columns, data, actions, getID, onRowClick } = props
+
+  const table = (
+    <table className={tableClassName || 'table table-hover'}>
+      <thead className={headerClassName || 'thead-light'}>
```

Comment:
We can put these values in the `defaultProps`

Reply:
✅ in aa4f804